gcp_variant_transforms/libs/processed_variant_test.py

```diff
@@ -108,14 +110,29 @@ def test_create_processed_variant_move_alt_info(self):
     self.assertEqual(proc_var.alternate_data_list, [alt1, alt2])
     self.assertFalse(proc_var.non_alt_info.has_key('A2'))
 
-  def _get_sample_variant_and_header_with_csq(self):
+  def _get_sample_variant_and_header_with_csq(self, additional_infos=None):
+    """Provides a simple `Variant` and `VcfHeader` with info fields
+
+    Args:
+      additional_infos: A list of tuples of the format (key, `Info`) to be added
+        to the `VcfHeader`.
+    """
+    # type:  (
     variant = self._get_sample_variant()
     variant.info['CSQ'] = ['A|C1|I1|S1|G1', 'TT|C2|I2|S2|G2', 'A|C3|I3|S3|G3']
-    header_fields = vcf_header_util.make_header({'CSQ': '.',
-                                                 'A1': '1', 'A2': 'A'})
-    header_fields.infos['CSQ'][
-        vcf_header_io.VcfParserHeaderKeyConstants.DESC] = (
-            'some desc Allele|Consequence|IMPACT|SYMBOL|Gene')
+    infos = OrderedDict([
```

Comment:
FYI, since you guys are Python 2.7 I would stay away from OrderedDict, as noted here:  

https://www.georgevreilly.com/blog/2017/02/21/OrderedDictInitialization.html